src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -574,6 +591,457 @@ export class MicroMetaTabView extends React.PureComponent {
+                tableHeader = (
+                    <div className="row summary-sub-header">
+                        <div className="col summary-title-column text-truncate">MetaData</div>
+                        <div className={columClassName + " summary-title-column"}>
+                            <DropdownButton title={matches[firstVisibleMatchIndex].Name} variant="outline-secondary btn-block text-left"
+                                size="md" className="w-100" onSelect={this.handleMatchSelection}>
+                                {
+                                    _.map(matches, function (match, idx) {
+                                        return (
+                                            <DropdownItem key={"match-" + idx} eventKey={idx}>
+                                                {match.Name || "-"}
+                                            </DropdownItem>);
+                                    })
+                                }
+                            </DropdownButton>
+                        </div>
+                    </div>
+                );
+            } else {
+                const showPrevBtn = matches.length > visibleMatchCount && firstVisibleMatchIndex > 0;
+                const showNextBtn = matches.length > visibleMatchCount && (firstVisibleMatchIndex + visibleMatchCount) < matches.length;
+
+                tableHeader = (
+                    <div className="row summary-sub-header">
+                        <div className="col col-lg-3 col-xl-3 summary-title-column text-truncate">MetaData {showPrevBtn ? this.prevNextButton(false) : null}</div>
+                        {_.map(visibleMatches, function (m, index) {
+                            return (
+                                <div className={columClassName + " summary-title-column text-truncate"}>
+                                    {m.Name}
+                                </div>
+                            );
+                        }, this)}
+                        {showNextBtn ? <div className="col-lg-1 col-xl-1 summary-title-column text-truncate">{this.prevNextButton(true)}</div> : null}
+                    </div>
+                );
+            }
+            
+            const schemaPropPairs = _.pairs(schema.properties);          
+
+            tableBody = _.map(_.keys(schema.subCategoriesOrder), function(subCategory){
+                const subCategoryProperties = _.filter(schemaPropPairs, function (spp) {
+                    const [, propItem] = spp;
+                    return propItem && propItem.category === subCategory;
+                });
+                const sectionProps = {
+                    subCategory, subCategoryProperties, matches: visibleMatches, tooltipLimit, columClassName, 
+                    collapsed: !!collapsedSections[subCategory]
+                };
+                return <CollapsibleSubCategory key={subCategory} {...sectionProps} toggleExpand={this.toggleExpand} />
```

Comment:
~~`this.toggleExpand` here refers to `undefined` because not using an arrow function (which would inherit `this`), a raw function 's `this` refers to itself~~ ah I see is using the _.map 2nd arg. Would still use an arrow function for clarity, though, and because might migrate away from underscore in future.

Reply:
(Updated/pushed)